crates/slumber_core/src/template.rs

```diff
@@ -942,7 +942,82 @@ mod tests {
 
         assert_err!(
             render!("{{chains.chain1}}", context),
-            "No response from prompt"
+            "No response from prompt/select"
+        );
+    }
+
+    #[rstest]
+    #[case::no_chains(vec!["foo!", "bar!"], Some("bar!"), "bar!")]
+    #[case::embedded_chain(vec!["{{chains.chain2}}", "bar!"], Some("foo!"), "foo!")]
+    #[case::all_chains(vec!["{{chains.chain2}}", "{{chains.chain3}}"], Some("baz!"), "baz!")]
```

Comment:
I don't think there's much value in these test cases, because we're just receiving an arbitrary string passed. Ideally we could validate that the given options are shown to the user, but the test machinery required for that is a lot more complicated so I would say we can just drop these two cases and chains 2+3.